Cargo.lock

```diff
@@ -484,6 +484,7 @@ version = "0.18.3"
 dependencies = [
  "Inflector",
  "anyhow",
+ "codegen_ebnf",
```

Comment:
for the `jco` commit, I suggest updating to the `nomic-enhancements-release` branch here:

https://github.com/NomicFoundation/jco/commit/c84e77fac0de001d57d54a09506c7ffde6398852

> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

Reply:
I believe the submodule is already at that commit.

Reply:
The PR is currently updating to [c84e77fa](
https://github.com/NomicFoundation/jco/commit/c84e77fac0de001d57d54a09506c7ffde6398852), which looks like it belongs to your fork. Did you mean to move to [51b184c4](https://github.com/NomicFoundation/jco/commit/51b184c43b35ad350989a55e4a8983ebf4861088), which belongs to the `nomic-enhancements-release` branch?